Makefile

```diff
@@ -77,12 +77,12 @@ unit-test:
 	@scripts/check_unit.sh
 
 .PHONY: bdd-test
-bdd-test: clean rp-adapter-rest-docker issuer-adapter-rest-docker mock-issuer-docker generate-test-config generate-test-keys
+bdd-test: clean did-method-cli rp-adapter-rest-docker issuer-adapter-rest-docker mock-issuer-docker generate-test-keys
 	@scripts/check_integration.sh
 
-.PHONY: generate-test-config
-generate-test-config:
-	@scripts/generate_test_config.sh
+.PHONY: did-method-cli
+did-method-cli:
```

Comment:
do we really have to pull the CLI into edge-adapter?

I see this has made the build take more time... would rather keep it as quick as possible.

Reply:
@Moopli ^^